aries_vcx/src/protocols/proof_presentation/verifier/state_machine.rs

```diff
@@ -70,6 +70,22 @@ pub enum RevocationStatus {
     NonRevoked,
 }
 
+fn build_verification_ack(thread_id: &str) -> PresentationAck {
+    PresentationAck::create().set_out_time().set_thread_id(thread_id)
+}
+
+fn build_starting_presentation_request(
+    thread_id: &str,
+    request_data: &PresentationRequestData,
+    comment: Option<String>,
+) -> VcxResult<PresentationRequest> {
+    PresentationRequest::create()
+        .set_out_time()
+        .set_id(thread_id.into())
+        .set_comment(comment)
+        .set_request_presentations_attach(request_data)
```

Comment:
I wonder whether these functions improve readability. I lean on the side they do - on one hand, they simplify the original code, on the other, they slightly obscure the actual behavior...

Reply:
The reason why I introduced these is mainly testing. It makes it much easier to test what kind of payloads we are building/sending